platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java

```diff
@@ -294,7 +294,11 @@ private void addConfigurationData(Service service, Configuration[] configs) {
                     .filter(metatype -> AttributeDefinition.PASSWORD == metatype.getType())
                     .forEach(metatype -> {
                         String passwordProperty = metatype.getId();
-                        propertiesTable.put(passwordProperty, "password");
+                        if (propertiesTable.get(passwordProperty) == null){
```

Comment:
I was having trouble with this because `propertiesTable.get(passwordProperty)` gives an `Object` so there is an incompatible type error with calling `StringUtils.isEmpty` on it

I could do something like the following but I'm not sure it improves simplicity?
```
String passwordProperty = metatype.getId();
String password = (propertiesTable.get(passwordProperty) == null) ?  "" : (propertiesTable.get(passwordProperty).toString();
if (StringUtils.isEmpty(password))) {
   propertiesTable.put(passwordProperty, "");
} else {
   propertiesTable.put(passwordProperty, "password");
}
```